design-documents/graph-ql/coverage/SubscribeEmailToNewsletter.graphqls

```diff
@@ -0,0 +1,7 @@
+type Mutation {
+    subscribeEmailToNewsletter(email: String!): SubscribeOutput @doc(description:"Adds an email into a newsletter subscription")
+}
+
+type SubscribeOutput {
+    status: Int! @doc(description: "Returns a status of subscription")
```

Comment:
Please introduce enum instead of int here.

Reply:
**done**